src/lib/components/colors/FormatGroup.svelte

```diff
@@ -0,0 +1,42 @@
+<script lang="ts">
+  import { inGamut } from 'colorjs.io/fn';
+  import type { PlainColorObject } from 'colorjs.io/types/src/color';
+
+  import Output from '$lib/components/colors/Output.svelte';
+  import type { ColorFormatId } from '$lib/constants';
+  import { ColorSpace } from '$lib/stores';
+  import { getSpaceFromFormatId } from '$lib/utils';
+
+  import ExternalLink from '../util/ExternalLink.svelte';
+
+  export let type: 'bg' | 'fg';
+  export let color: PlainColorObject;
+  export let name: string;
+  export let formats: ColorFormatId[];
+
+  $: gamutSpace = ColorSpace.get(
+    getSpaceFromFormatId(formats[0] as ColorFormatId),
```

Comment:
To be clear, this is always going to check `inGamut` against the first space in the list from `FORMAT_GROUPS`, and display that space name in the out-of-gamut warning.

1. Just to confirm, is the gamut always identical between spaces in each grouping?
2. It doesn't seem ideal for the displayed name to be implicitly reliant on the ordering of the formats (e.g. if `hsl` comes first, the warning text changes). Could we add an explicit mapping of what space we check against for each group or something like that?